### PR TITLE
Ninjum patch 1

### PR DESCRIPTION
**Pull request by ninjum**
various corrections

### CODE REVIEW

File: android/assets/screens/credits.gdxui [hunk 1]
```diff
@@ -71,6 +71,9 @@
                 <Label>Basque:</Label>
                 <Label>Josu Igoa</Label>
                 <Label/>
+                <Label>Galego:</Label>
```
Comment (agateau) [inline android/assets/screens/credits.gdxui:74]: The language should be in English here.

```suggestion
                <Label>Galician:</Label>
```

File: android/assets/ui/languages.xml [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <Language id="es" name="Español" fontSet="xolonium-kwajong"/>
     <Language id="eo" name="Esperanto" fontSet="xolonium"/>
     <Language id="eu" name="Euskara" fontSet="xolonium-kwajong"/>
+    <Language id="gl" name="Galician" fontSet="xolonium-kwajong"/>
```
Comment (agateau) [inline android/assets/ui/languages.xml:10]: Here it's the other way around: the language should be translated.
```suggestion
    <Language id="gl" name="Galego" fontSet="xolonium-kwajong"/>
```

**Review by agateau — CHANGES_REQUESTED** [review]
Hi @ninjum! Thanks a lot for this contribution. Just two minor remarks and it can be merged.

**Comment by ninjum** [comment]
corrections made, is everything correct already?

**Review by agateau — APPROVED** [review]

**Comment by agateau** [comment]
Looks good, thanks a lot!

**Comment by agateau** [comment]
One last thing: I would need an email address to notify you when I am preparing a new release, so that you can update your translation. Can you send me your email address to mail@agateau.com?

**Pull request merged**